src/assets/components/ns-card/standard-pill.html

```diff
@@ -0,0 +1,8 @@
+<ns-card image="/placeholder-16x9-640x360px.webp">
```

Comment:
Please omit the image in this example:

```suggestion
<ns-card>
```